.travis.yml

```diff
@@ -17,13 +24,17 @@ matrix:
     - php: 7.1
     - php: 7.2
     - php: hhvm
+    - stage: release
+      if: branch = master
```

Comment:
should it be on master, or on tags ?

Reply:
we used to do it on master but tags is indeed a better idea